ReSwift/CoreTypes/StoreSubscriber.swift

```diff
@@ -23,9 +23,7 @@ extension StoreSubscriber {
     // swiftlint:disable:next identifier_name
     public func _newState(state: Any) {
         if let typedState = state as? StoreSubscriberStateType {
-            DispatchQueue.main.sync {
-                newState(state: typedState)
-            }
+            newState(state: typedState)
```

Comment:
This should be optional; Always calling `newState` in the main thread may be unnecessary, or even unwanted.